src/Components/Panels/PanelMenu.tsx

```diff
@@ -56,21 +57,23 @@ interface PanelMenuState extends SceneObjectState {
  */
 export class PanelMenu extends SceneObjectBase<PanelMenuState> implements VizPanelMenu, SceneObject {
   constructor(state: Partial<PanelMenuState>) {
-    super(state);
+    super({ ...state, addExplorationsLink: state.addExplorationsLink ?? true });
     this.addActivationHandler(() => {
-      const viz = sceneGraph.getAncestor(this, VizPanel);
+      const viz = findObjectOfType(this, (o) => o instanceof VizPanel, VizPanel);
 
       this.setState({
-        addToExplorations: new AddToExplorationButton({
+        explorationsButton: new AddToExplorationButton({
           labelName: this.state.labelName,
           fieldName: this.state.fieldName,
           frame: this.state.frame,
         }),
       });
 
-      // @todo rewrite the AddToExplorationButton
-      // Manually activate scene
-      this.state.addToExplorations?.activate();
+      if (this.state.addExplorationsLink) {
+        // @todo rewrite the AddToExplorationButton
```

Comment:
Why must it be rewritten?

Reply:
This is for Sven from the previous PR that introduced the explorations link to the PanelMenu, the component was originally written to be a button in the panel header